pkg/providers/version/version.go

```diff
@@ -69,39 +68,32 @@ func NewDefaultProvider(kubernetesInterface kubernetes.Interface, cache *cache.C
 }
 
 func (p *DefaultProvider) Get(ctx context.Context) (string, error) {
```

Comment:
Since this will always succeed, let's drop the error.
```suggestion
func (p *DefaultProvider) Get(ctx context.Context) string {
```